lib/response.js

```diff
@@ -142,6 +142,55 @@ module.exports = (function () {
             }
             console.log(`Error saving state: ${err}\n${err.stack}`);
             this.context.fail(err);
+        },
+        ':play': function (title, url) {
+            if (this.isOverridden()) {
+                return;
+            }
+
+            // If last time the audio ends on its own, reset the audio attributes and play from the top of the radio list.
+            if (this.attributes[':playbackFinished']) {
+                this.attributes[':index'] = 0;
+                this.attributes[':offsetInMilliseconds'] = 0;
+                this.attributes[':playbackIndexChanged'] = true;
+                this.attributes[':playbackFinished'] = false;
+            }
+
+            // If the user does not dedicate a audio to play, play the first audio on the list.
+            if (!title) {
+                title = this.handler.audioData[this.attributes[':playOrder'][this.attributes[':index']]].title;
+            }
+            if (!url) {
+                url = this.handler.audioData[this.attributes[':playOrder'][this.attributes[':index']]].url;
+            }
+
+            // Update the index of the audio.
+            var flag = false;
+            for(var i = 0; i < this.handler.audioData.length; i++){
+                if (this.handler.audioData[i].url === url && this.handler.audioData[i].title === title) {
+                    this.attributes[':index'] = this.attributes[':playOrder'].indexOf(i);
+                    flag = true;
+                    break;
+                }
+            }
+            if(!flag) {
+                console.log("The provided url and title are not in the audioData.");
+                this.context.succeed(true);
```

Comment:
this will cause a JSON parsing error from the Alexa service. Only valid JSON objects can be returned

please also fix the other instances in this file